### DISCUSSION TITLE
Run Mac OS X in Docker with near-native performance?

### DISCUSSION THREAD

**Discussion by ash-hashtag** [sickcodes/Docker-OSX#562] [General]
I can't get it to work on anywhere ` near native performance `

but I'm not complaining, it is more than enough, it is work-able ( just for compiling my flutter projects ) , and it's fine.

I tried to do most of the mentioned optimizations except for labeled "Do at your own risk", I've checked my bios, and my cpu does support virtualization
but still it is no where near native, mouse delays sometimes, keyboard input lags.

What maybe I'm missing?
Or the "near native thing" just a buzz word, and everyone experiencing the same?

**Comment by Coool** [comment]
"Run Mac OS X in Docker with near-native performance!".
What does it mean? Is it some minor performance penalty or not compared to running on Linux using QEMU with KVM? What are benefits running on Docker?

Related issue:
- https://github.com/sickcodes/Docker-OSX/discussions/641
